### PR TITLE
ext65 refactor

### PR DESCRIPTION
**Pull request by Xelus22**
## Description

Refactor EXT65 so that it is easier to update and maintain in future and shares the same keymaps.

I'm not sure if this should hit develop or master.

Note: includes #10226 change

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Refactor  EXT65

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/aeboards/ext65/readme.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
-# Ext65Rev2
+EXT65
+===
```
Comment (fauxpark) [inline keyboards/aeboards/ext65/readme.md:2]:
```suggestion
# EXT65
```

**Review by fauxpark — APPROVED** [review]

File: keyboards/aeboards/ext65/config.h [hunk 2]
```diff
@@ -0,0 +1 @@
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/aeboards/ext65/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/aeboards/ext65/ext65.h [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+#include "quantum.h"
+
+#ifdef KEYBOARD_aeboards_ext65_rev1
+    #include "rev1.h"
+#endif
+
+#ifdef KEYBOARD_aeboards_ext65_rev2
+    #include "rev2.h"
+#endif
```
Comment (drashna) [inline keyboards/aeboards/ext65/ext65.h:11]:
```suggestion
#if defined(KEYBOARD_aeboards_ext65_rev1)
    #include "rev1.h"
#elif defined(KEYBOARD_aeboards_ext65_rev2)
    #include "rev2.h"
#endif
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**